src/OrganizationsSDKAdapter.test.js

```diff
@@ -0,0 +1,59 @@
+import {isObservable} from 'rxjs';
+
+import OrganizationsSDKAdapter from './OrganizationsSDKAdapter';
+import createMockSDK, {mockSDKOrganization} from './mockSdk';
+
+describe('Organizations SDK Adapter', () => {
+  let mockSDK;
+  let organizationsSDKAdapter;
+  let organizationID;
+
+  beforeEach(() => {
+    mockSDK = createMockSDK();
+    organizationsSDKAdapter = new OrganizationsSDKAdapter(mockSDK);
+  });
+
+  afterEach(() => {
+    mockSDK = null;
+    organizationsSDKAdapter = null;
+  });
+
+  describe('getOrg()', () => {
+    beforeEach(() => {
+      organizationID = 'organizationID';
+      organizationsSDKAdapter.fetchOrganization = jest.fn(
+        () => Promise.resolve(mockSDKOrganization),
+      );
+    });
+
+    test('returns an observable', () => {
+      expect(isObservable(organizationsSDKAdapter.getOrg(organizationID))).toBeTruthy();
+    });
+
+    test('emits organization details on subscription', (done) => {
+      organizationsSDKAdapter.getOrg(organizationID).subscribe(
+        (organization) => {
+          expect(organization).toMatchObject({
+            ID: 'organizationID',
+            name: 'Cisco Systems, Inc.',
+          });
+          done();
+        },
+      );
+    });
+  });
+
+  describe('getorg() errors', () => {
```

Comment:
No need to add this extra describe, unless there was some special setup, but that doesn't seem to be the case